Source/JS API/TabBar.swift

```diff
@@ -16,10 +16,12 @@ import JavaScriptCore
 @objc public class TabBar: ViewControllerChild, TabBarJSExport {
     
     public func hide() {
+        //log(.Debug, "") // provide breadcrumbs
         parentViewController?.tabBarController?.tabBar.hidden = true
     }
     
     public func show() {
+        //log(.Debug, message: "") // provide breadcrumbs
```

Comment:
can we just remove this as well as the call on line 19 instead of committing commented out code?